File: project/ovp/server/src/main/java/com/mobigen/ovp/glossary/client/dto/glossary/Glossary.java
package com.mobigen.ovp.glossary.client.dto.glossary;

import com.mobigen.ovp.common.openmete_client.dto.Owner;
import lombok.Data;

import java.util.List;

@Data
public class Glossary {
    private String id;
    private String name;
    private String displayName;
    private String description;
    private String fullyQualifiedName;
    private Owner owner;
    private List<Object> tags;
}

File: project/ovp/frontend/ovp/type/glossary.ts
export interface Glossary {
  id: string;
  name: string;
  displayName: string;
  description: string;
  fullyQualifiedName: string;
  owner: Owner;
  //TODO response type 정의
  tags: object;
}
export interface Term {
  id: string;
  displayName: string;
  description: string;
  fullyQualifiedName: string;
  name: string;
}
export interface Activity {
  cardStyle: string;
  createdBy: string;
  headerMessage: string;
  id: string;
  message: string;
  updatedAt: string;
  updatedBy: string;
  fieldOperation: string;
  entitySpecificInfo: object;
}
export interface Owner {
  id: string;
  type: string;
  name: string;
  fullyQualifiedName: string;
  displayName: string;
}
